lib/core/macros.nim

```diff
@@ -237,6 +237,10 @@ else: # bootstrapping substitute
     else:
       n.strValOld
 
+when defined(nimSymImplTransform):
+  proc getImplTransformed*(symbol: NimNode): NimNode {.magic: "GetImplTransf", noSideEffect.}
+    ## for a typed proc returns the AST after transformation pass
```

Comment:
To this day, there is not a single test for this proc in the entire test suite.

Reply:
fixed in 14924